data/campaigns/Liberty/scenarios/07_Glory.cfg

```
@@ -381,7 +381,7 @@ One score and eight years ago, in 70 YW, the King of Wesnoth and the Lord of Ele
         [/message]
         [message]
             type=Lancer
-            message= _ "At your order, my liege."
+            message= _ "At your order, my Liege."
```

I don't get why this is capitalized…?

---

Me neither, I'll fix it.
```suggestion
            message= _ "At your order, my liege."
```